app/services/description_validator.rb

```diff
@@ -2,23 +2,56 @@
 
 # Validate the descriptive metadata spreadsheet
 class DescriptionValidator
-  def initialize(csv)
+  def initialize(csv, bulk_job: false)
+    @csv = csv
     @headers = csv.headers
+    @bulk_job = bulk_job # indicates if validating from a bulk job
     @errors = []
   end
 
   attr_reader :errors
 
   def valid?
+    validate_headers
+    validate_rows
+    errors.empty?
+  end
+
```

Comment:
refactor to avoid rubocop complexity complaints about this `valid?` method getting too big